css/style.css

```diff
@@ -9,400 +14,633 @@ body {
+  display: flex;
+  width: 475px;
+  align-items: center;
+  justify-content: space-between;
 }
 
 .button {
-    font-style: normal;
-    font-weight: 500;
-    font-size: 13px;
-    line-height: 0;
-    border: 1.5px solid #000;
-    background-color: #fff;
-   
-     
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+  line-height: 0;
+  border: 1.5px solid #000;
+  background-color: #fff;  
 }
 
 .new-сollection__main-button {
-    width: 203px;
-    height: 44px; 
+  width: 203px;
+  height: 44px; 
 }
 
 .link {
-    font-style: normal;
-    font-weight: 500;
-    font-size: 13px;
-    line-height: 0px;
-    color: #000;
-    text-decoration: none;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+  line-height: 0px;
+  color: #000;
+  text-decoration: none;
 }
 
 .new-сollection__main-link {
-width: 172px;
-height: 0px;
+  width: 172px;
+  height: 0px;
 }
 
 .new-сollection__social {
-    display: flex;
-    align-items: center;
-    justify-content: flex-start;
-    top: 752px;
-    position: absolute;
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  top: 752px;
+  position: absolute;
 }
 
 .instagram {
-    margin-right: 15px;
+  margin-right: 15px;
 }
 
 .vk {
-    margin-right: 15px;
+  margin-right: 15px;
 }
 
 .fb {
-    margin-right: 500px;
+  margin-right: 500px;
 }
 
 .line {
-    position: absolute;
-    top: 12px;
-    right: 12px;
+  position: absolute;
+  top: 12px;
+  right: 12px;
 }
 .img {
-    position: relative;
+  position: absolute;
+  top: 3%;
+  right: 0;
 }
+
 .bag__title-bold {
-    font-style: normal;
-    font-weight: bold;
-    font-size: 40px;
-    line-height: 41px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 41px;
 }
 
 .bag__title {
-    font-style: normal;
-    font-weight: bold;
-    font-size: 16px;
-    line-height: 16px;
-    letter-spacing: .07em;
-    opacity: .5;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 16px;
+  letter-spacing: .07em;
+  opacity: .5;
 }
 
 .bag-text{
-    font-style: normal;
-    font-weight: normal;
-    font-size: 15px;
-    line-height: 21px;
-    opacity: .5;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 21px;
+  opacity: .5;
 }
 
 .product {
-    font-style: normal;
-    font-weight: 500;
-    font-size: 13px;
-    line-height: 16px; 
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+  line-height: 16px; 
 }
 
 .bag-title-bold {
-    font-style: normal;
-    font-weight: bold;
-    font-size: 40px;
-    line-height: 41px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 41px;
 }
 
 .bag-title {
-    font-style: normal;
-    font-weight: bold;
-    font-size: 16px;
-    line-height: 16px;
-    letter-spacing: .07em;
-    opacity: .5;  
+  font-style: normal;
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 16px;
+  letter-spacing: .07em;
+  opacity: .5;  
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Love-metal_LOCO/pullRequest?prid=5478856)